svgnative/src/SVGDocumentImpl.cpp

```diff
@@ -69,33 +66,21 @@ void SVGDocumentImpl::TraverseSVGTree()
 
     if (!HasAttr(rootNode, "viewBox"))
     {
-        std::array<float, 4> bounds{};
         if (HasAttr(rootNode, "x"))
-            bounds[0] = SVGDocumentImpl::ParseLengthFromAttr(rootNode, "x", LengthType::kHorrizontal);
+            mViewBox[0] = SVGDocumentImpl::ParseLengthFromAttr(rootNode, "x", LengthType::kHorrizontal);
         if (HasAttr(rootNode, "y"))
-            bounds[1] = SVGDocumentImpl::ParseLengthFromAttr(rootNode, "y", LengthType::kVertical);
+            mViewBox[1] = SVGDocumentImpl::ParseLengthFromAttr(rootNode, "y", LengthType::kVertical);
         if (HasAttr(rootNode, "width"))
-            bounds[2] = SVGDocumentImpl::ParseLengthFromAttr(rootNode, "width", LengthType::kHorrizontal);
+            mViewBox[2] = SVGDocumentImpl::ParseLengthFromAttr(rootNode, "width", LengthType::kHorrizontal);
         if (HasAttr(rootNode, "height"))
-            bounds[3] = SVGDocumentImpl::ParseLengthFromAttr(rootNode, "height", LengthType::kVertical);
-        mViewBox.clear();
```

Comment:
It might be a bit unsafe to rely on the previous state of mViewBox if one of these attributes (x, y, width, height) is not present. But I see that TraverseSVGTree is called only once and right after the constructor so we could leave as is.